web_src/js/features/repo-code.ts

```diff
@@ -110,19 +111,21 @@ function showLineButton() {
 }
 
 export function initRepoCodeView() {
-  if (!document.querySelector('.code-view .lines-num')) return;
-
-  let selRangeStart: string;
-  addDelegatedEventListener(document, 'click', '.lines-num span', (el: HTMLElement, e: KeyboardEvent) => {
-    if (!selRangeStart || !e.shiftKey) {
-      selRangeStart = el.getAttribute('id');
-      selectRange(selRangeStart);
-    } else {
-      const selRangeStop = el.getAttribute('id');
-      selectRange(`${selRangeStart}-${selRangeStop}`);
-    }
-    window.getSelection().removeAllRanges();
-    showLineButton();
+  registerGlobalInitFunc('initRepoCodeView', (el: HTMLElement) => {
+    if (!el.querySelector('.lines-num')) return;
+
+    let selRangeStart: string;
+    addDelegatedEventListener(el, 'click', '.lines-num span', (el: HTMLElement, e: KeyboardEvent) => {
```

Comment:
I do not think the fix is right. You will have duplicate "addDelegatedEventListener" calls and add duplicate "click" event listeners.

Reply:
After some testing, I think it breaks the "blame" page.